csrc/device_lower/pass/alias_memory.cpp

```diff
@@ -743,15 +763,23 @@ class AllocationInfoMap : private kir::IrVisitor {
 
   void handle(kir::ForLoop* for_loop) final {
     auto loop_info = scope_map_.getLoopScopeInfo(for_loop);
-    current_stack_.push_back(loop_info);
+    if (!for_loop->iter_domain()->isParallelized()) {
```

Comment:
Should we instead use `for_loop->isTrivial()`? There are other trivial loops not generated in codegen, for example, vectorization loop, and we should handle all of them equivalently.

Reply:
Much better. Thanks.